device/rsvp/rsvpInterface.yaml

```diff
@@ -0,0 +1,81 @@
+components:
+  schemas:
+    Rsvp.Interface:
+      x-status: under-review
+      description: >-
+        Configuration for RSVP Interface.
+      type: object
+      required: [ipv4_name,neighbor_ip]
+      properties:
+        ipv4_name: 
+          description: >-
+            The globally unique name of the IPv4 interface connected to the DUT.            
```

Comment:
What is ipv4_name? Basically, it is a reference to the ipv4 connected interface that the user has configured earlier in the " "ipv4_addresses" object. Isn't it? So can it be - _This name should be the "name" field of the "ipv4_addresses" on top which you want to configure this interface"_

Reply:
Added a line for indicating the required match to 'name' field. But for now not modifying to second person ( your...) . This is because in vendor documentation or openconfig/yang model documentation, we could not find any such usage.